src/cmd/top/joblist_pane.c

```diff
@@ -243,12 +250,13 @@ void joblist_pane_query (struct joblist_pane *joblist)
                              "job-list.list",
                              0,
                              0,
-                             "{s:i s:i s:i s:i s:[s,s,s,s,s,s,s]}",
+                             "{s:i s:i s:i s:i s:[s,s,s,s,s,s,s,s]}",
                              "max_entries", win_dim.y_length - 1,
                              "userid", FLUX_USERID_UNKNOWN,
                              "states", FLUX_JOB_STATE_RUNNING,
                              "results", 0,
                              "attrs",
+                               "queue",
```

Comment:
you have this in the `flux-mini: remove -q short option for --quiet` commit, I guess a merge conflict.

Reply:
Yeah accidentally committed some WIP stuff. Oops